cmd/gapit/video.go

```diff
@@ -25,6 +25,7 @@ import (
 	"image/png"
 	"io"
 	"os"
+	fp "path/filepath"
```

Comment:
why the alias? do we have any style guide when to use aliases?

Reply:
Because we use "filepath" elsewhere as a variable name.

Reply:
Ack.